src/Components/Book.js

```diff
@@ -1,17 +1,28 @@
+/* eslint-disable camelcase */
 import { PropTypes } from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/Books/Bookslice';
+import { checkStatus } from '../redux/categories/categoriesslice';
 
 function Book({ book }) {
-  const { title, author } = book;
+  const dispatch = useDispatch();
+  const {
+    item_id,
+    title,
+    author,
+  } = book;
   return (
-    <div className="Books">
+    <div id={item_id} className="Books">
       <div className="B1">
         <h4>Action</h4>
         <h2>{title}</h2>
         <h3>{author}</h3>
         <div className="comps">
           <h3>comments</h3>
-          <h3>remove</h3>
+          <button className="btn" type="button" onClick={() => dispatch(removeBook({ item_id }))}>remove</button>
+          <button className="btn" type="button" onClick={() => dispatch(checkStatus())}>check status</button>
```

Comment:
- [ ] I think this checkStatus dispatch here is meant for the categories component. When I visit the categories component and click the check status button, you should dispatch an action to the store and fetch the "under construction" string from the store and display it on the page. Kindly do that.

  ![image](https://user-images.githubusercontent.com/54848363/223759244-fae4784d-7ac1-4e46-b034-41c393eec2e7.png)